files/install_puppet.ps1

```diff
@@ -303,8 +303,8 @@ try {
   Write-Log "Waiting for pxp-agent processes to stop"
   Get-Process -Name "pxp-agent" -ErrorAction SilentlyContinue | ForEach-Object {
     if ($_) {
-      # wait on each process for 2 minutes (120000 milliseconds)
-      if (!$_.WaitForExit(120000)){
+      # wait on each process for 8 minutes (420000 milliseconds)
+      if (!$_.WaitForExit(420000)){

```

Comment:
Can this be parameterized so we don't need to do releases to change it?

Reply:
@joshcooper - This could most likely be parameterized. I will generate a separate PR for that. Will post the link to that PR once that has been completed.

Reply:
@joshcooper - my previous commit has been updated. I removed the old commit hard coding the value at 8 minutes and parameterized the module.